content/components.md

```diff
@@ -4,13 +4,13 @@ Providers are a fundamental concept in Nest. Many of the basic Nest classes may
 
 <figure><img src="/assets/Components_1.png" /></figure>
 
-In the previous chapter, we built a simple `CatsController`. Controllers should handle HTTP requests and delegate more complex tasks to **providers**. Providers are plain JavaScript classes with an `@Injectable()` decorator preceding their class declaration.
+In the previous chapter, we built a simple `CatsController`. Controllers should handle HTTP requests and delegate more complex tasks to **providers**. Providers are plain JavaScript classes that are declared as Providers in a [module](/modules.md).
```

Comment:
```suggestion
In the previous chapter, we built a simple `CatsController`. Controllers should handle HTTP requests and delegate more complex tasks to **providers**. Providers are plain JavaScript classes that are declared as Providers in a [module](/modules).
```

Reply:
the link does not lead to the modules page

Reply:
Thanks, fixed!